launch

```diff
@@ -97,14 +100,27 @@ CONFIG = {
             'rpi3': {
                 'binary': 'qemu-mimiker-aarch64',
                 'options': [
+                    '-kernel', '{kernel}',
+                    '-initrd', '{initrd}',
                     '-machine', 'raspi3',
                     '-smp', '4',
                     '-cpu', 'cortex-a53'],
                 'network_options': [],
                 'uarts': [
                     dict(name='/dev/cons', port=RandomPort(), raw=True)
                 ]
-            }
+            },
+            'pc': {
+                'binary': 'qemu-mimiker-x86_64',
+                'options': [
+                    '-cpu', 'qemu64',
+                    '-drive', ('if=pflash,format=raw,'
+                               'file=../OVMF.fd'),
+                    '-drive', 'format=raw,file=disk.img,id=hd0',
+                    '-device', 'VGA'],
```

Comment:
FTTB, I've removed the VGA device form the configuration.

Reply:
This was done because GRUB uses text mode by default, but UEFI graphics protocol doesn't support that. So to avoid a warning (and thereby an error), I have added the VGA device and a tag to instruct GRUB to use the framebuffer instead of text mode.

Reply:
But it isn't a problem in the current state of the PR, so the tag and device are gone for now.

Reply:
Please note that the above snippet is outdated and doesn't reflect the current state of this file in this PR. Check "Files Changed" for the accurate version.

Reply:
Oh. You are right. For unknown for me reason github didn't mark that file as outdated.